addons/website_event/static/src/js/tours/event_tour.js

```diff
@@ -26,7 +26,7 @@ EventAdditionalTourSteps.include({
                 position: 'bottom',
                 run: 'drag_and_drop iframe #o_wevent_event_main_col',
             }, {
-                trigger: 'button[data-action="save"]',
+                trigger: ".o_we_save_container button",
```

Comment:
This (and same for the 2 comments that follow) is different from the previous commit. Here I couldn't simply replace the step with a call to the tour util because the contents differ (the sentence showed to the users).
And if I don't replace the save step with the util, I can't update it, as `.o_we_save_container` does not exist in the previous commit.
So I keep it as is (`[data-action="save"]`) and update it in the main commit instead.